cumulus/node/README.md

```diff
@@ -7,103 +7,66 @@
 
 Start a farmer:
 
 ```bash
-$ ./target/release/subspace-farmer wipe && ./target/release/subspace-farmer farm
+$ ./target/release/subspace-farmer wipe && ./target/release/subspace-farmer farm --reward-address REWARD_ADDRESS --plot-size 10G
 ```
 
 Now the primary node should be producing blocks.
 
-2. Run an executor as an authority node.
-
-Ensure the bootnode for the primary node is correct and run this command to start an executor:
+2. Run another executor running as a full node.
 
 ```bash
 $ ./target/release/subspace-node \
     --chain dev \
     -d db1 \
-    --log=trace \
-    --bootnodes "/ip4/127.0.0.1/tcp/30333/p2p/PRIMARY_PEER_ID" \
-    --port 30343 \
-    --ws-port 9977 \
-    -- \
-        --alice \
-        --port 40333 \
-        --log=txpool=trace,gossip=trace \
-        --rpc-port 8845 \
-        --ws-port 8846 \
-
-```
-
-The log for running the secondary node will be prefixed as `[Secondarychain]`, you should see it start to produce blocks as well.
-
-```
-...
-2022-04-01 09:34:56.010 TRACE tokio-runtime-worker cirrus::executor: [Secondarychain] Origin deduplicated extrinsics extrinsics=[]
-2022-04-01 09:34:56.011 TRACE tokio-runtime-worker cirrus::executor: [Secondarychain] Shuffled extrinsics shuffled_extrinsics=[]
-2022-04-01 09:34:56.013 TRACE tokio-runtime-worker txpool: [Secondarychain] Pruning transactions: []
-2022-04-01 09:34:56.013 DEBUG tokio-runtime-worker txpool: [Secondarychain] Starting pruning of block BlockId::Hash(0x7089dfba167eeb17b361126db248dbb0bb2a1ded9f485e20c517b3a8f5800604) (extrinsics: 0)
-2022-04-01 09:34:56.013 DEBUG tokio-runtime-worker txpool: [Secondarychain] Pruning at BlockId::Hash(0x7089dfba167eeb17b361126db248dbb0bb2a1ded9f485e20c517b3a8f5800604)
-2022-04-01 09:34:56.013 TRACE tokio-runtime-worker txpool: [Secondarychain] Pruning at BlockId::Hash(0x7089dfba167eeb17b361126db248dbb0bb2a1ded9f485e20c517b3a8f5800604). Resubmitting transactions.
-2022-04-01 09:34:56.013  INFO tokio-runtime-worker substrate: [Secondarychain] ✨ Imported #18 (0x7089…0604)
-2022-04-01 09:34:56.013 DEBUG tokio-runtime-worker cirrus::executor: [Secondarychain] Trace root calculated for #0x7089…0604 trace=[0xe99ff5a2f994e4832ffc093c10b4d1d294a401b0bbd9d52db7523716d9864140, 0x63cf7a793cc3c20f68cd3d683ba9effe7d87245ddf3b0c52ba9bac43eef7b653] trace_root=[23, 87, 107, 20, 223, 81, 204, 197, 221, 24, 70, 36, 204, 4, 23, 135, 162, 250, 135, 179, 131, 83, 169, 73, 9, 72, 122, 237, 90, 139, 239, 25]
-...
-
-```
-
-3. Run another executor as a full node.
-
-```bash
-$ ./target/release/subspace-node \
-    --chain dev \
-    -d db2 \
     --bootnodes "/ip4/127.0.0.1/tcp/30333/p2p/PRIMARY_PEER_ID" \
     --port 30443 \
-    --ws-port 9987
+    --ws-port 9987 \
     -- \
-        --alice \
+        -- \
         --port 40233 \
         --log=cirrus=trace,txpool=trace,gossip=trace \
         --rpc-port 8745 \
```

Comment:
There are two `--` here and extra `\` at the very end.

Also do I understand correctly that 2 executor chains will not be connected together, only primary chains are (I don't see bootstrap nodes for executors)?

Reply:
Double `--` is necessary as the arguemnts with a single `--` fail to be parsed. The trailing `\` can be removed.

Yes, I deliberately make the executor nodes not connected for now, it's one of the problems I'd love to discuss on the next protocol sync call as well: the sync strategy between the executors, do we still want to use the traditional syncing or just process every single primary block to produce the final local state? I'm not sure about that.

Reply:
Try adding `--alice` before second `--port`. Then it works. Not sure why it doesn't without it, doesn't make sense to me.

But it does work with `-- --`. Weird...